config.yaml

```
@@ -4,13 +4,8 @@
 # If your charm does not require configuration options, delete this file entirely.
 #
 # See https://juju.is/docs/config for guidance.
-
 options:
-  # An example config option to customise the log level of the workload
-  log-level:
-    description: |
-      Configures the log level of gunicorn. 
-
-      Acceptable values are: "info", "debug", "warning", "error" and "critical"
-    default: "info"
+  external-hostname:
+    default: ""
+    description: The hostname of the ingress.
```

I might be confused here - would this be the hostname of the ingress that we want to configure it as, or the hostname of the ingress we want to supply to the charm?
```suggestion
    description: The hostname of the ingress to configure as.
```

---

This will be the hostname we want to supply to the ingress. As a first step we'll implement a path routing similar to the `traefik-k8s` charm where a hostname is configured on the gateway and the url of the application will be something like `https://<hostname>/<path>`